tiatoolbox/wsicore/wsireader.py

```diff
@@ -3081,6 +3092,13 @@ def _canonical_shape(self, shape):
                 Shape in YXS order.
 
         """
+        # toilet roll multiplex OME TIFF
```

Comment:
@vqdang  Please can you provide a reference to this comment?

Reply:
I believe he is referring to the planar configuration of the TIFF file, see https://awaresystems.be/imaging/tiff/tifftags/planarconfiguration.html